packages/kbn-test/src/functional_test_runner/lib/config/run_check_ftr_configs_cli.ts

```diff
@@ -66,24 +68,45 @@ export async function runCheckFtrConfigsCli() {
           return false;
         }
 
-        if (file.match(/mocks.ts$/)) {
+        // No FTR configs in /packages/
+        if (file.match(/\/packages\//)) {
+          return false;
+        }
+
+        if (file.match(/(mock|mocks).ts$/)) {
           return false;
         }
 
         const fileContent = readFileSync(file).toString();
 
-        if (fileContent.match(/(testRunner)|(testFiles)/)) {
+        if (
+          // explicitly define 'testRunner' or 'testFiles'
+          fileContent.match(/(testRunner)|(testFiles)/) ||
+          // export default createTestConfig
+          fileContent.match(/export\s+default\s+createTestConfig/) ||
+          // export default async function ({ readConfigFile }: FtrConfigProviderContext)
+          // async function config({ readConfigFile }: FtrConfigProviderContext)
+          // export default async function (ftrConfigProviderContext: FtrConfigProviderContext)
+          fileContent.match(
+            /(?:export\s+default\s+)?async\s+function(?:\s+\w+)?\s*\(\s*(?:\{\s*readConfigFile\s*\}|\w+)\s*(?::\s*FtrConfigProviderContext\s*)?\)/
```

Comment:
Can we name this?  That's a lot of reading lol

Reply:
yes, I can put it into variable with comment